package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "csmm",
-  "version": "1.19.0",
+  "version": "1.19.1",
```

this get bumped on release, so lets leave it as 0 for now.
```suggestion
  "version": "1.19.0",
```

---

sure